kernel/build/build_deb

```diff
@@ -88,11 +88,17 @@ echo "*****"
 # Clean the git repo
 git clean -fxd
 
+# Install build dependencies
+sudo apt-get install -y bc
+
 # Prep the config
 echo "Preparing the kernel config"
-cp "$(dirname $0)/kernel-config-deb.basic" .config
+wget https://raw.githubusercontent.com/ceph/ceph-build/master/kernel/build/kernel-config-deb.basic -O .config
```

Comment:
Instead of using wget here you can use multiple ``scm`` entires in the config to have jenkins also clone ceph-build as well as ceph-client. I think that's the approach we should take here.

This job is a good example of that: https://github.com/ceph/ceph-build/blob/master/shaman-pull-requests/config/definitions/shaman-pull-requests.yml#L1

Then in the build script for that job you can see how I use different paths for ceph-build vs shaman.

https://github.com/ceph/ceph-build/blob/master/shaman-pull-requests/build/build#L9